images/gitness/config/template.apko.yaml

```diff
@@ -0,0 +1,36 @@
+contents:
+  packages:
+    - gitness
+    - git
+    - busybox
```

Comment:
as you specified these packages in extra_packages, do you still need to specify them in here too?